i18n/fr.yaml

```diff
@@ -5,74 +5,62 @@ twitter_description:
 privacy_action:
-  other: "Read our sample privacy policy"
+  other: "Lire notre politique de confidentialité"
 three_parts:
-  other: "COVID Shield is made up of three parts: a mobile app, a server, and a
-    web-based results portal."
+  other: "La solution COVID Shield est formée par trois composantes : une application mobile, un serveur et un portail web de résultats."
 mobile_app_title:
-  other: "Mobile app (iOS and Android)"
+  other: "Application mobile (iOS et Android)"
 mobile_app_content:
-  other: "The mobile app runs in the background and requires no user interaction after onboarding. It uses Bluetooth to collect and share random IDs with nearby phones with COVID Shield installed. If a user tests positive for COVID-19, they can choose to anonymously share their data so others can be notified of possible exposure. Sharing of random IDs is voluntary and only possible with a positive test result confirmed by a health care professional. COVID Shield periodically downloads the shared random IDs from the server and compares them on each user’s device to determine if a possible exposure has occurred."
+  other: "L'application mobile fonctionne en arrière-plan et n'exige aucune interaction avec l'utilisateur après l'intégration. Elle utilise le Bluetooth pour recueillir et partager des identifiants aléatoires avec les téléphones situés à proximité, sur lesquels COVID Shield est installé. Si un utilisateur est testé positif au COVID-19, il peut choisir de partager anonymement ses données de manière que d'autres puissent être avisés d'une possible exposition. Le partage des identifiants aléatoires se fait sur une base volontaire et n'est possible que si le résultat positif du test a été confirmé par un professionnel de santé. COVID Shield télécharge périodiquement les identifiants aléatoires partagés depuis le serveur et les compare sur l'appareil de chaque utilisateur pour déterminer si une exposition possible a eu lieu."
 server_title:
-  other: "Server"
+  other: "Serveur"
 server_content:
-  other: "The server securely collects and stores random IDs uploaded from the mobile app after a confirmed positive test result. It also generates unique temporary codes which grant permission for people with a positive COVID-19 test result to upload their random IDs. The server is flexible enough to be deployed at various levels of government."
+  other: "Le serveur recueille et stocke de façon sécurisée les identifiants aléatoires importés depuis l'application mobile suite à un résultat de test positif confirmé. Il génère également des codes temporaires uniques qui accordent aux individus testés positifs au COVID-19 l'autorisation de mettre en ligne leurs identifiants aléatoires. Le serveur est suffisamment flexible pour un déploiement à différents ordres de gouvernement."
```

Comment:
1- "testé positif" is an anglicism, we use "reçoit un résultat de test positif"
2- we say "à la" COVID-19, and not "au" COVID-19
So this paragraph becomes: 
`Le serveur recueille et stocke de façon sécurisée les identifiants aléatoires importés depuis l'application mobile suite à un résultat de test positif confirmé. Il génère également des codes temporaires uniques qui accordent aux individus ayant reçu un résultat de test positif à la COVID-19 l'autorisation de mettre en ligne leurs identifiants aléatoires. Le serveur est suffisamment flexible pour un déploiement à différents ordres de gouvernement.`